scripts/lib/install_opencv.sh

```diff
@@ -1,13 +1,34 @@
 #
-# Install opencv and its dependencies
+# Install opencv and related dependencies
 
 install_opencv() {
```

Comment:
We dont need comments for each command unless you want to group commands under one comment

Reply:
removed comments